### PR TITLE
Ts 668

### PR DESCRIPTION
**Pull request by tomskikh**
#668

### CODE REVIEW

File: src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  /**
+    * Returns an average execution time of some method (in nanoseconds)
+    *
+    * @return an average execution time of some method (in nanoseconds)
+    */
+  def average: Long = time.sum / time.length
+
+  /**
+    * Returns an smallest execution time of some method (in nanoseconds)
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala]: the smallest

File: src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    * Returns an largest execution time of some method (in nanoseconds)
+    *
+    * @return an largest execution time of some method (in nanoseconds)
+    */
+  def max: Long = time.max
+
+  /**
+    * Returns an 95% confidence interval of execution time of some method (in nanoseconds)
+    *
+    * @return an largest execution time of some method (in nanoseconds)
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala]: the largest

Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala]: fixed

File: src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala (hunk 1 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/ExecutionTimeMeasurement.scala]: fixed

**Pull request merged**